ckanext/canada/plugins.py

```diff
@@ -88,6 +88,7 @@ class CanadaDatasetsPlugin(SchemingDatasetsPlugin):
     p.implements(p.IDatasetForm, inherit=True)
     p.implements(p.IPackageController, inherit=True)
     p.implements(p.IBlueprint)
+    p.implements(p.IActions)
```

Comment:
all the other plugins in this file implement IActions, maybe add it to one of those instead?

Reply:
moved to the Public plugin class